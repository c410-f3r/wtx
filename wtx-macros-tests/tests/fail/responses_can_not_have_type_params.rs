#[wtx_macros::pkg(data_format(json), id(FooId))]
mod pkg {
  #[pkg::req_data]
  struct Req;

  #[pkg::res_data]
  struct Res<T>(T);
}

fn main() {
}
