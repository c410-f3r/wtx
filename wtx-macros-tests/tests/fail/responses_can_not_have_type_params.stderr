error: Responses can have at most one lifetime. Types aren't supported
 --> tests/fail/responses_can_not_have_type_params.rs:7:10
  |
7 |   struct Res<T>(T);
  |          ^^^

error: aborting due to 1 previous error

